tests/ClientTest.php

```diff
@@ -34,6 +34,11 @@ public function testSetupFormatters()
         $formatter = $client->getFormatter('testConfig');
         $this->assertTrue($formatter instanceof UrlEncodedFormatter);
         $this->assertEquals(PHP_QUERY_RFC3986, $formatter->encodingType);
+
+        $unrecognizedFormat = 'unrecognizedFormat';
```

Comment:
It is better to move it to separate test-method such as `testSetupUnrecognizedFormat`.